samples/stages.js

```diff
@@ -0,0 +1,31 @@
+import http from "k6/http";
+import { check } from "k6";
+
+
+/*
+ * Stages (aka ramping) is how you, in code, specify the ramping of VUs.
+ * That is, how many VUs should be active and generating traffic against
+ * the target system at any specific point in time for the duration of
+ * the test.
+ * 
+ * The following stages configuration will result in up-flat-down ramping
+ * profile over a 20s total test duration.
+ */ 
+
+export let options = {
+    stages: [
+        // Ramp-up from 1 to 5 VUs in 10s
+        { target: 5, duration: "10s" },
```

Comment:
Really pedantic thing: I think putting these in the order "duration, target" is more readable, since you can omit target, but not duration. It provides a consistent vertical column to scan.